docs/README.md

```diff
@@ -335,6 +335,23 @@ Configures the git username and email to associate commits with the provided ide
 
 The two vars in the previous snippet are [workflow configuration variables](https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/) that can be created at organization level and shared across different repositories.
 
+### free-runner-disk-space
```

Comment:
maybe rename to `free-hosted-runner-disk-space` given those paths are specific to [hosted runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners)?